### DISCUSSION TITLE
Get focused input tag

### DISCUSSION THREAD

**Discussion by IvanIhnatsiuk** [kirillzyusko/react-native-keyboard-controller#163] [Ideas]
Hello @kirillzyusko and thank you for the great library. 

I just want to suggest how you can improve your library In the case of using the forms. 

P.S My inspiration came from the react-native-keyboard-aware-scrollview library :)

So, it would be really cool if we can get some information which input is currently focused, for example we can get absolute coordinates or view tag and calculates it's via react-native-reanimated measure function. 
So, when we get these values we can determine exactly should we scroll to this component or not. 

what do you think about it?

**Comment by kirillzyusko** [comment]
Hi @IvanIhnatsiuk 

I'll look into this 👍 Thank you for the suggestion! 😎

**Comment by kirillzyusko** [comment]
Close as [1.6.0](https://github.com/kirillzyusko/react-native-keyboard-controller/releases/tag/1.6.0) was published to npm
